druid/src/widget/flex.rs

```diff
@@ -927,6 +948,17 @@ impl From<f64> for FlexParams {
     }
 }
 
+enum Child<T> {
+    Fixed(WidgetPod<T, Box<dyn Widget<T>>>, Option<CrossAxisAlignment>),
```

Comment:
I personally find that once I hit two or so fields I really prefer struct variants over enum variants.

Reply:
oh wow why do people use tuple variants when struct variants exist!!! this is so much better!!!!!!!!

Reply:
I remember, you cant easily ignore fields...